dvc/command/daemon.py

```diff
@@ -24,10 +25,12 @@ def run(self):
 
 class CmdDaemonAnalytics(CmdDaemonBase):
     def run(self):
-        from dvc.analytics import Analytics
+        from dvc import analytics
+
+        report = self.args.target
 
-        analytics = Analytics.load(self.args.target)
-        analytics.send()
+        analytics.send(report)
+        os.remove(report)
```

Comment:
Let's move this to analytics. Analytics generated it - analytics will delete it, no need to spread the logic across two files.

Reply:
@efiop , this is happening under `CmdDaemonAnalytics`, it is related to analytics :slightly_smiling_face: 

I can wrap it under the analytics module like:
```python
def remove(report):
    os.remove(report)
```

Reply:
@mroutis File creation happens in dvc/analytics.py but deletion happens in dvc/command/analytics.py . Feels weird.

Reply:
@efiop , moving `os.remove` under `analytics.remove` will do it for you?

Could you explain what feels weird about it?

Reply:
> moving os.remove under analytics.remove will do it for you?

and then calling it here? It is no different from what you have right now. 

> moving os.remove under analytics.remove will do it for you?

You are creating the file in analytics.py, when calling analytics.collect_and_send, right? So why do you need to delete the file created in analytics.py in command/daemon.py, where it is pretty much out of context with the former code? Why not do that after you've sent the report in the same code context that it was created in? It feels like you are spreading the logic across multiple locations, which is pretty weird to me. That being said, this is a very minor detail, so I won't mind much keeping it where it is right now, if we don't agree.

Reply:
Your van remove in .send()